Fix css issue of image block full width variante

iFlameing opened the pull request:
Fixes #115

danalvrz reviewed (approved):
LGTM, just a couple questions. Why is the breakpoint `1470px` specifically? And where can I find the design spec for this change?

iFlameing commented:
@danalvrz We have a full-width snap at 1440px. I have added another 30px because that is the currently available padding-left for full width. That's why I added another 30px to make it looks nice and consistent.
